docs/admin/logging/overview.md

```diff
@@ -130,6 +131,10 @@ By default, Docker uses the first 12 characters of the container ID to tag log m
 Refer to the [log tag option documentation](log_tags.md) for customizing
 the log tag format.
 
+`syslog-format` configures syslog protocol version to use when logging,
+Allowed value is rfc5424 in order to use RFC5424 syslog protocol. 
+Default is RFC3164 incompatible syslog protocol.
```

Comment:
@thaJeztah  this rfc issue is slightly more complicated, see the default (which was an original default) is not exactly RFC3164 (hence i wrote the incompatible :))
So basically there are 3 choices we can add (or any combination of them) 
1) rfc5424
2) RFC3164
3) default (which is syslog defauly but rfc3164 incompatible). 
Should i add those options ?